applications/source/debezium-source/src/test/resources/docker/mssql/init-with-signalling.sql

```diff
@@ -0,0 +1,93 @@
+-- Create the test database
+CREATE DATABASE testDB;
```

Comment:
This file gets renamed during dockerfile copy to init.sql, correct? If so, maybe we can remove the other `init.sql` that was copied over as it seems to be unused. Then just name this `init.sql`?